src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,190 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.exception.InvalidDataException;
+import core.basesyntax.model.handler.OperationHandler;
+import core.basesyntax.model.handler.impl.BalanceOperationHandler;
+import core.basesyntax.model.handler.impl.PurchaseOperationHandler;
+import core.basesyntax.model.handler.impl.ReturnOperationHandler;
+import core.basesyntax.model.handler.impl.SupplyOperationHandler;
+import core.basesyntax.model.service.OperatorService;
+import core.basesyntax.model.service.ReaderService;
+import core.basesyntax.model.service.ReportService;
+import core.basesyntax.model.service.WriterService;
+import core.basesyntax.model.service.impl.OperatorServiceImpl;
+import core.basesyntax.model.service.impl.ReaderServiceImpl;
+import core.basesyntax.model.service.impl.ReportServiceImpl;
+import core.basesyntax.model.service.impl.WriterServiceImpl;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class FruitShopTest {
```

Comment:
[Don’t keep all tests in a single class](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#dont-test-main-class)